Dockerfile

```diff
@@ -1,32 +1,28 @@
-FROM ghcr.io/linuxserver/baseimage-kasmvnc:alpine317
+FROM ghcr.io/linuxserver/baseimage-kasmvnc:arch
 
 # set version label
 ARG BUILD_DATE
 ARG VERSION
-ARG DARKTABLE_VERSION
+ARG DIGIKAM_VERSION
```

Comment:
Harmless typo

Reply:
I have been found out, lets hope the maintainers of https://github.com/linuxserver/docker-digikam don't find out we stole their code.